package.json

```diff
@@ -78,7 +78,7 @@
     "vue-plugin-helper-decorator": "^0.0.12",
     "vue-property-decorator": "^8.5.1",
     "vue-template-compiler": "2.6.12",
-    "vue-test-utils-helpers": "https://github.com/bcgov/vue-test-utils-helpers.git",
+    "vue-test-utils-helpers": "github:bcgov/vue-test-utils-helpers",
```

Comment:
I'm still curious/worried about why this declaration looks like this but the one in business-filings-ui looks like this:
`"vue-test-utils-helpers": "git+https://github.com/bcgov/vue-test-utils-helpers.git",`

What do you think? Merge it and see if this works?

Reply:
OK, this now looks the same as in Filings UI.